udata-front-plugins-helper/package.json

```diff
@@ -19,15 +19,18 @@
     "publish-stable": "npm publish --access public",
     "publish-next": "npm run publish-stable -- --tag next"
   },
+  "overrides": {
+    "tough-cookie": "^4.1.3"
```

Comment:
Not sure I totally understand how overrides work, but can't we remove the `overrides` once package-lock has been updated?

Reply:
I put it here to match the fix we did on [udata-front](https://github.com/etalab/udata-front/blob/master/package.json#L23) and so it doesn't rely on package-lock not being erased.